zkevm-circuits/src/evm_circuit/witness.rs

```diff
@@ -324,7 +325,7 @@ impl Bytecode {
     pub fn table_assignments<'a, F: FieldExt>(
         &'a self,
         randomness: F,
-    ) -> impl Iterator<Item = [F; 4]> + '_ {
+    ) -> impl Iterator<Item = [F; 5]> + '_ {
```

Comment:
I think you can change the return type and simplify the logic

```suggestion
    ) -> Vec<[F; 5]> {
```